### PR TITLE
Setup grid UI

### PR DESCRIPTION
**Pull request by dnewh**
This PR sets up a bare-bones grid UI Canopy project with a minimal Grid theme.

### CODE REVIEW

File: grid-ui/.gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (vaporos) [inline grid-ui/.gitignore]: Move this to the top-level -- we don't want .gitignore files in subdirectories.

File: grid-ui/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.<br /> Your app is
+ready to be deployed!
+
+See the section about
+[deployment](https://facebook.github.io/create-react-app/docs/deployment) for
+more information
+
+## License
```
Comment (vaporos) [inline grid-ui/README.md]: This license section should be removed as it duplicates information at the top of the repository.

File: grid-ui/.gitignore (hunk 1 above)
Comment (dnewh) [reply grid-ui/.gitignore]: removed

File: grid-ui/README.md (hunk 2 above)
Comment (dnewh) [reply grid-ui/README.md]: removed

**Pull request merged**